charts/reporting-operator/templates/custom-resources/prom-queries/persistentvolumeclaim-status.yaml

```diff
@@ -0,0 +1,11 @@
+apiVersion: metering.openshift.io/v1alpha1
+kind: ReportPrometheusQuery
+metadata:
+  name: persistentvolumeclaim-status
```

Comment:
probably want `persistentvolumeclaim-status-phase` since there can potentially be other status's

Reply:
Sure